plugins/org.erlide.ui/plugin.xml

```diff
@@ -2148,7 +2148,7 @@
    <extension
          point="org.eclipse.ui.preferencePages">
       <page
-            category="org.erlide.ui.preferences.compiler"
+            category="org.erlide.ui.preferences"
```

Comment:
:+1: (This changes the menu item to Erlang > Dialyzer instead of Erlang > Compiler > Dialyzer.)

Reply:
Updated top comment